### ISSUE TITLE
[Toolbar UI Experiments][Voice over] "X" button in address bar not announced by screen reader when address bar is in edit mode

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/firefox-ios#25503]
## Steps to reproduce
1. Open the app.
1. Tap the address bar to activate edit mode.
1. Make sure there is text inside the address bar.
1. Turn on voice over
1. Swipe through the elements within the address bar.

### Expected behavior
The "X" button should be accessible and announced

### Actual behavior
The "X" (clear text) button is not announced.

### Device & build information
* Device: 6s(15.7) , iPhone 15 (17.6)
* Build version: 137 (52491)

┆Issue is synchronized with this [Jira Bug](https://mozilla-hub.atlassian.net/browse/FXIOS-11701)

**Comment by data-sync-user** [comment]
➤ Andres Furlan commented:

Test it in nightly

**Comment by data-sync-user** [comment]
➤ Alina Moldovan commented:

Verified as fixed using v137.1 (53321) and iPhone 16 Pro (8.3.2)

[^ScreenRecording_04-08-2025 14-24-42_1.MP4]
